docs/api/graphql_queries.md

```diff
@@ -181,6 +181,72 @@ Response:
 }
 ```
 
+### Get Matrix Field Type
+
+To get a Matrix Field Type, use `product` field.
+
+The example below shows Product Content Type that has two fields:
+
+- `name`: `ezstring`
+- `features`: `ezmatrix` with two columns: `name` and `price`
+
+Fields of that type return a list of `ProductFeaturesRow`.
```

Comment:
This is only valid for the `product` content type with the `features` field. The types that are returned and used for input are named after the type and field: `{TypeIdentifier}{FieldIdentifier}Row` and `{TypeIdentifier}{FieldIdentifier}RowInput`.